set.mm

```diff
@@ -152852,6 +152853,17 @@ a function (ordinarily on a subset of ` CC ` ) and produces a new
     ( cmnf cxr wcel cc0 clt wbr csgn cfv c1 cneg wceq mnfxr mnflt0 sgnn mp2an )
     ABCADEFAGHIJKLMANO $.
 
+$( The signum of an extended real is zero if and only if that extended real
+     equals zero.  (Contributed by Simon Dold, 26-Sep-2022.) $)
+  sgn0iff0 $p |- ( A e. RR* -> ( ( sgn ` A ) = 0 <-> A = 0 ) ) $=
```

Comment:
Sorry, but this theorem is already existing in set.mm: see ~ sgn0bi, contributed by TA in 2018 (in his mathbox). Furthermore, TA's proof is shorter, so his one would be preferred.

Regarding the naming: We usually use the suffix "bi"  or "b" for biconditionals, not the label fragment "iff".